command.go

```diff
@@ -411,6 +411,102 @@ func (cmd *StringCmd) Float64() (float64, error) {
 	return strconv.ParseFloat(cmd.val, 64)
 }
 
+func (cmd *StringCmd) Scan(vv interface{}) error {
```

Comment:
I think it is 50/50. E.g. it will simplify first 4 cases, but in the 5-10th cases I will have to define `var n int64`. So I will trade return for var declaration, which does not make much difference for me...